docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "2"
+services:
+    mongo:
+        image: mongo
+        volumes:
+            - ./data:/data/db
+        ports:
+            - "27017:27017"
+    web:
+        build:
+            context: .
```

Comment:
Additionally this is good for dev, but we'll also need an image only one for prod deploys.